.github/workflows/verify-page.yml

```diff
@@ -1,20 +1,14 @@
 name: Verifies gh-page is up and running
 
 on:
-  push:
-    branches:
-      - main
-      - playwright-base
+  schedule:
+    - cron: "0 * * * 1-5"
```

Comment:
That's midnight, mon - fri?

Reply:
every 0 minute evyer hour => 
<img width="1057" alt="image" src="https://github.com/AxisCommunications/fluent-components/assets/105843747/b6a28fe3-35dd-4e00-acbe-a163afe6d7dc">

Reply:
Seems a bit excessive...?

Reply:
It will be adjusted in next iteration, i think its fine :D. I was planning to set it every 10 min first. 
I kind of just want to monitor it a  while then adjust.
Is is ok or u got a suggested schedule i should adjust to? :)